libs/booking/root/feature-shell/.eslintrc.json

```diff
@@ -9,7 +9,7 @@
         "plugin:@angular-eslint/template/process-inline-templates"
       ],
       "parserOptions": {
-        "project": ["libs/booking/feature-shell/tsconfig.*?.json"]
+        "project": ["libs/booking/root/feature-shell/tsconfig.*?.json"]
```

Comment:
These paths where wrong causing the problems, saying that that projectSettings were missing.
Fyi, there are still a few errors because it's linting json files.

Reply:
Thanks. These paths are probably not updated by the `move` generator, or perhaps I moved the projects manually.